config/manifests/bases/awx-operator.clusterserviceversion.yaml

```
@@ -456,13 +456,16 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:resourceRequirements
-      - description: The PostgreSQL init container is not used when an external DB
-          is configured
-        displayName: PostgreSQL Init Container Resource Requirements
-        path: postgres_init_container_resource_requirements
+      - description: Sets permissions on the /var/lib/postgresql/data for postgres container using an init container (not Openshift)
```

Is it `/var/lib/postgresql/data` or `/var/lib/pgsql/data` ?
```suggestion
      - description: Sets permissions on the /var/lib/pgsql/data for postgres container using an init container (not Openshift)
```